packages/backend-defaults/config.d.ts

```diff
@@ -82,6 +82,16 @@ export interface Config {
           };
         };
 
+    /**
+     * Options used by the default auditor service.
+     */
+    auditor?: {
+      /**
+       * The lowest log level that will be recorded. Defaults to "medium".
+       */
+      logLevel?: 'low' | 'medium' | 'high' | 'critical';
```

Comment:
Just my two cents I think there's two things to consider which are verbosity and severity (log level)

Maybe you do want everything logged, but you just want it to be on a lower verbosity.

Wondering how we could maybe tackle that 🤔 💭

Reply:
Hmm, yep I agree, and I think taking that even further I don't think we want a log level config for the default auditor at all. All we need is really config for a log level mapping right? and then we leave the actual log level filtering to the logger service. For example something like this:

```
backend:
  auditor:
    severityLogLevels:
      low: info
      critical: error
```

The above config would leave the log level for medium and high as they are by default, but set the log level of low severity to info, and critical severity to error.

What do you think @schultzp2020?